DEVELOPER_GUIDE.md

```diff
@@ -279,19 +279,19 @@ Please also check out the message pack examples on the [msgpack-c GitHub repo](h
 
 ### Plugin API
 
-Each plugin is a shared object which is [loaded into Fluent Bit](https://github.com/fluent/fluent-bit/blob/1.3/src/flb_plugin.c#L70) using dlopen and dlsym.
```

Comment:
I chose to pin these to a specific version on purpose. As the code changes the line numbers will change, so unless the link is updated, it will eventually become wrong.

Since this is for educational purposes I think it's already to pin to a specific version in time.

Reply:
Sure thing. I changed this to fill in the dead end links only then.